### ISSUE TITLE
Script to “six-ify” source

### ISSUE THREAD

**Issue by benjaminp** [benjaminp/six#8]
Originally reported by: **Zearin (Bitbucket: [Zearin](https://bitbucket.org/Zearin), GitHub: [Zearin](https://github.com/Zearin))**

----------------------------------------

**Wouldn’t it be cool if six had a script in the spirit of `2to3`?  **

Imagine if this (invented) example were possible:
    
    $ python -m "six" spam.py eggs.py food/*.py
    
    …Adding "import six" to source…
    Success!
    
    …Converting "isinstance" and "issubclass" to six constants for:
        …six.class_types…
        Success!
        
        …six.integer_types…
        Success!
        
        …six.string_types…
        Success!
    
    …Converting "print" to "six.print_"…
        Success on 38 of 40 occurrences; couldn’t convert the following:
        
        spam.py:135 : Ambiguous print arguments.  Please fix manually.
        spam.py:202 : Ambiguous print arguments.  Please fix manually.
        
        Comments labelled "# @SIXME" have been inserted to mark these locations.
    
    

---- 

**Basically:**

Six is awesome, and there are plenty of cases where existing could needs the scrutiny of a human to determine how to integrate six.  But there are plenty of cases where it’s a no-brainer: 

* uses of `isinstance` or `issubclass`
* `print`
* module renames (as seen in //[[http://packages.python.org/six/#module-six.moves|Renamed modules and attributes compatibility]]// of the documentation.) 

Following `2to3`’s example, the script should allow fine-grained control to opt-in or opt-out of any particular conversions.  

----------------------------------------
- Bitbucket: https://bitbucket.org/gutworth/six/issue/8
